hedera-node/hedera-schedule-service-impl/build.gradle.kts

```diff
@@ -32,4 +32,5 @@ dependencies {
     testImplementation(testFixtures(project(":hedera-node:hedera-mono-service")))
     testImplementation(testLibs.bundles.mockito)
     compileOnly(libs.spotbugs.annotations)
+    implementation(libs.swirlds.virtualmap)
```

Comment:
I am not sure . I wasn't able to import `ScheduleVirtualValue` to `ScheduleStore`. Because `SCheduleStore` should create a model object for merkle types. I don't see any other way without this.

Reply:
For now I think you will need this. With my new shiny `states` implementations, you won't need this dependency anymore